Yosemite/Yosemite/Stores/ProductStore.swift

```diff
@@ -512,6 +513,24 @@ private extension ProductStore {
         }
         storageProduct.addToTags(NSOrderedSet(array: storageTags))
     }
+
+    /// Updates, inserts, or prunes the provided StorageProduct's downloads using the provided read-only Product's downloads
+    ///
+    func handleProductDownloads(_ readOnlyProduct: Networking.Product, _ storageProduct: Storage.Product, _ storage: StorageType) {
```

Comment:
This should be tested 👍

Reply:
As discussed on slack. Will commit these changes on a separate PR targeting `develop`